services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteLeg.java

```diff
@@ -45,6 +46,17 @@ public static Builder builder() {
   @Nullable
   public abstract Double duration();
 
+  /**
+   * The estimated travel time from one waypoint to another including delay (road repair,
+   * traffic jam, etc.).
```

Comment:
```suggestion
   * The typical travel time for traversing this RouteLeg. There's a delay along the RouteLeg if you subtract this durationTypical() value from the RouteLeg duration() value and the resulting difference is greater than 0. The delay is because of any number of real-world situations (road repair, traffic jam, etc).
```